internal/model/user.go

```diff
@@ -0,0 +1,65 @@
+package model
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+type User struct {
+	ID       string
+	Username string
+	Email    string
+	Password string
+}
+
+type CreateUserReq struct {
+	Username string `json:"username"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
```

Comment:
domain models shouldn't have any kind of struct tags. 

For request/response models you have to create dedicated model in handler package